src/EA.Weee.RequestHandlers.Tests.DataAccess/Search/FetchOrganisationSearchResultsForCache/FetchOrganisationSearchResultsForCacheDataAccessTests.cs

```diff
@@ -731,8 +731,8 @@ public async Task FetchOrganisations_OrganisationWithDirectRegistrant_OneIncompl
 
             List<DirectRegistrant> directRegistrants = new List<DirectRegistrant>()
             {
-                new Domain.Producer.DirectRegistrant(organisationComplete, null, null, null),
-                new Domain.Producer.DirectRegistrant(organisationIncomplete, null, null, null),
+                new Domain.Producer.DirectRegistrant(organisationComplete),
```

Comment:
Does this not complain about missing parameters?

Reply:
No, created a constructor that doesnt need them... Probably this sprint will make the contact and address non nullable fields, as we didnt have all screens couldn't do that so this will get updated.